src/utils/helpers.py

```diff
@@ -89,34 +89,11 @@ def convert_formtype(formtype_value: str) -> str:
         return None
 
 
-def convert_formtype(formtype_value: str) -> str:
-    """Convert the formtype to a standardised format.
-
-    Args:
-        formtype_value (str): The value to standardise.
-
-    Returns:
-        str: The standardised value for formtype.
-    """
-    if pd.notnull(formtype_value):
-        formtype_value = str(formtype_value)
-        if formtype_value == "1" or formtype_value == "1.0" or formtype_value == "0001":
-            return "0001"
-        elif (
-            formtype_value == "6" or formtype_value == "6.0" or formtype_value == "0006"
-        ):
-            return "0006"
-        else:
-            return None
-    else:
-        return None
-
-
```

Comment:
Have you removed this cos it's not used and we have another function doing the same thing in construction?